src/components/Profile/components/PersonalInfoList/components/UpdatePhoneDialog/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import MaskedInput from 'react-text-mask';
+import EditIcon from '@material-ui/icons/Edit';
+import {
+  IconButton,
+  FormControl,
+  Input,
+  InputLabel,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core';
+import userService from 'services/user';
+import user from 'services/user';
```

Comment:
One of these should go- we only want one reference to the same file.

Reply:
You are right!